WoWsShipBuilder.Web/Components/DispersionPlot.razor

```diff
@@ -39,7 +39,7 @@
 @code {
 
     [Parameter]
-    public List<DispersionEllipse> Ships { get; set; } = default!;
+    public Dictionary<Guid, DispersionEllipse>  Ships { get; set; } = default!;
```

Comment:
Don't initialize with "default!" when there is no EditorRequired attribute.
Also check if using ImmutableDictionary may be the right choice here. Ideally, parameters are immutable types